vmware-event-router/hack/README.md

```diff
@@ -0,0 +1,76 @@
+# Deploy vCenter Event Broker Application to existing Kubernetes Cluster
+
+For customers with an existing Kubernetes Cluster, you can deploy the underlying components that make up the vCenter Event Broker Appliance. The instructions below will guide you in downloading the required files and using the `create_k8s_config.sh` shell script to aide in deploying the VEBA K8s application.
+
+The script will prompt users for the required input and  automatically setup and deploy both OpenFaaS and the VMware Event Router giving you a similar setup like the vCenter Event Broke Appliance.
+
+## Pre-Req:
+* Deployment access to your K8s Cluster using kubectl
+* Outbound connectivity or access to private  registry from the K8s Cluster to download the required containers to deploy OpenFaaS and VMware Event Router
+
+## Install:
+
+Step 1 - Clone the OpenFaaS to your local system
+
+```
+git clone https://github.com/openfaas/faas-netes
+```
+
+Step 2 - Change into the `faas-netes` directory and checkout version `0.9.2` which has been tested with VEBA
+
+
+```
+cd faas-netes
+git checkout 0.9.2
+```
+
+Step 3 - Download the `create_k8s_config.sh` script and ensure it has executable permission (`chmod +x create_k8s_config.sh`).
```

Comment:
It would be good to specify that this script needs to be run from within the OpenFaaS git folder you check out above. Alternatively, you can use curl to retrieve this file and pipe it to a SHELL (which most installers pulling from Github do these days as well).

Reply:
Actually, the script runs outside of the OpenFaaS directory. Corrected the instructions and in case of debugging, I prefer the user has the script on hand